### PR TITLE
Created single comment view 

### PR DESCRIPTION
**Pull request by stanislav-slavov**
ref #321
ref #445
ref #446

### CODE REVIEW

**Comment by mradkov** [comment]
Screenshot 2020-04-10 at 17 15 57

The comment ID in the URL is mistaken. Url should be: `/tip/TIP_ID/comment/COMMENT_ID`

File: src/router.js [hunk 1]
```diff
@@ -17,13 +18,33 @@ const guardTipComments = (to, from, next) => {
...
+
 const routes = [
   {
     path: '/',
     name: 'home',
     component: TipsList,
     meta: { title: 'Home' },
   },
+  {
+    path: '/tip/:id/comment/:id',
```
Comment (mradkov) [inline src/router.js:39]:
```suggestion
    path: '/tip/:tipId/comment/:id',
```

Comment (venimus) [reply src/router.js:39]: can we request an API entrypoint for comments? this does not seem logical if comments has unique ID, you don't need the tipId to request the entity data

Comment (mradkov) [reply src/router.js:39]: I think we should keep the routes like this, for easier filtering later on in the comment section and also easier associating on-chain and off-chain comments with the specific tip.

Comment (venimus) [reply src/router.js:39]: Also I think this should be done with `route children`

File: src/views/SingleCommentView.vue [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+  },
+  beforeDestroy() {
+    clearInterval(this.interval);
+  },
+  methods: {
+    getAvatar(address) {
+      const userImage = Backend.getProfileImageUrl(address);
+      return userImage || this.avatar;
+    },
+    // async sendTipComment() {
```
Comment (venimus) [inline src/views/SingleCommentView.vue:140]: can you clean up these, or it is still WIP

File: src/router.js (hunk 1 above)
Comment (mradkov) [reply src/router.js:39]: https://github.com/aeternity/tipping-community-backend/blob/superhero/routes/README.md

File: src/views/SingleCommentView.vue (hunk 2 above)
Comment (stanislav-slavov) [reply src/views/SingleCommentView.vue:140]: This should be implemented - sending comment to another comment

**Comment by thepiwo** [comment]
lets close this for now as there hasn't been any progress the last week, feel free to repopen

**Comment by mradkov** [comment]
The backend now supports the nested comments. Lets revive this.

**Comment by davidyuk** [comment]
The support of nested comments implemented in the wallet in this PR: https://github.com/aeternity/superhero-wallet/pull/259

**Comment by mradkov** [comment]
Screenshot 2020-05-05 at 0 17 17

**Comment by mradkov** [comment]
Can we also visualize the reply better as it is in the designs:

Screenshot 2020-05-05 at 0 18 35

File: src/router.js [hunk 3]
```diff
@@ -17,13 +18,33 @@ const guardTipComments = (to, from, next) => {
   }
 };
 
+const guardCommentView = (to, from, next) => {
+  if (to.name === 'comment'
+      && typeof to.params.id !== 'undefined'
+      && typeof to.params.tipId !== 'undefined') {
+    next();
+  } else {
+    next('/');
```
Comment (venimus) [inline src/router.js:27]: why not 404?

File: src/components/tipRecords/TipComment.vue [hunk 4]
```diff
@@ -1,39 +1,45 @@
...
-            <Avatar :address="comment.author" />
-            <span
-              v-if="userChainName"
-              class="chain__name"
+          <span
+            class="user-display"
+            @click.stop
+          >
+            <router-link
+              :to="'/user-profile/' + comment.author"
```
Comment (venimus) [inline src/components/tipRecords/TipComment.vue:19]: IMHO best is to pass params as object

File: src/router.js (hunk 3 above)
Comment (stanislav-slavov) [reply src/router.js:27]: We don't have 404 page. I guess this a matter of a different user-story. Our maintenance page is there to show that the site is down.

**Pull request merged**